rust/azure_iot_operations_protocol/src/common/payload_serialize.rs

```diff
@@ -137,7 +163,7 @@ impl PayloadSerialize for BypassPayload {
         };
         Ok(BypassPayload {
             content_type: ct,
-            format_indicator: format_indicator.clone(),
+            format_indicator: *format_indicator,
```

Comment:
If it implements `Copy` now, this is still more complex than it needs to be - don't even bother taking a reference in the params and simplify it further.